image/scripts/pod_setup.sh

```diff
@@ -1,11 +1,9 @@
 #!/bin/bash
 
-su -s /bin/bash -c "${CONTAINER_APP_ROOT}/app/bin/bundle exec rake --trace db:migrate RAILS_ENV=$RAILS_ENV" "${CONTAINER_APP_USERNAME}"
-su -s /bin/bash -c "${CONTAINER_APP_ROOT}/app/bin/bundle exec rake assets:precompile RAILS_ENV=$RAILS_ENV" "${CONTAINER_APP_USERNAME}" 2>&1 | sed 's/^/asset-build: /'
+"${CONTAINER_APP_ROOT}/app/bin/bundle" exec rake --trace db:migrate
+"${CONTAINER_APP_ROOT}/app/bin/bundle" exec rake assets:precompile 2>&1 | sed 's/^/asset-build: /'
 
 if [ -n "${DISCOURSE_USE_S3}" ] && [ "${DISCOURSE_USE_S3}" == "true" ]; then
 	echo "Running migration to S3..."
-	su -s /bin/bash -c "${CONTAINER_APP_ROOT}/app/bin/bundle exec rake s3:upload_assets RAILS_ENV=$RAILS_ENV" "${CONTAINER_APP_USERNAME}" 2>&1
-else
-	echo "Not running migration to S3 (DISCOURSE_USE_S3=${DISCOURSE_USE_S3})."
+	"${CONTAINER_APP_ROOT}/app/bin/bundle" exec rake s3:upload_assets 2>&1
```

Comment:
Should we add prefix to this command output like previous commands?
Something like `| sed 's/^/upload-assets: /'`